.pre-commit-config.yaml

```diff
@@ -39,4 +39,4 @@ repos:
 -   repo: https://github.com/myint/rstcheck
     rev: master
     hooks:
-    -   id: rstcheck
+    -   id: rstcheck
```

Comment:
```suggestion
    -   id: rstcheck

```

Reply:
@norlandrhagen, I think adding this newline will get us back to even with `master` for this file